listener/inventory_processor.py

```diff
@@ -126,8 +127,21 @@ def _format_vmaas_req(self, package_list: [], repo_list: [], basearch: str, modu
             vmaas_request["basearch"] = basearch
         if releasever:
             vmaas_request["releasever"] = releasever
+        if organization:
+            vmaas_request["organization"] = organization
```

Comment:
wouldn't it cause re-evaluation of all systems? I'm not sure how it is done here in vuln but I suppose vmaas_json digest is used to determine if the system should be evaluated or not